src/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js

```diff
@@ -11,12 +11,51 @@ import { useTranslation } from 'react-i18next';
 import { makeStyles } from '@material-ui/core';
 
 const useStyles = makeStyles(theme => ({
+  root: {
+    height: '100%',
+    width: '100%',
+    position: 'relative'
+  },
   divContainer: {
     height: '100%',
     width: '100%'
+  },
+  errorContainer: {
+    height: '50px',
+    width: '100%',
+    position: 'absolute',
+    textAlign: 'center',
+    padding: '5px 0',
+    backgroundColor: theme.palette.error.dark,
```

Comment:
I can't get to find the color used for wrong scenario names message, but I'd like both of them to have the same centralized value if you can. Maybe that's already the case.

Reply:
The color for the scenario name errors is probably a default color right now.
When we want to unify the colors, using the `theme.palette.error.[dark/light/main]` value will be the proper way to do that :+1:

Reply:
LGTM then :)